client/signup/steps/business-confirmation/index.tsx

```diff
@@ -0,0 +1,188 @@
+/**
+ * External Dependencies
+ */
+import React, { Component } from 'react';
+import { localize, LocalizeProps } from 'i18n-calypso';
+import { connect } from 'react-redux';
+import { pickBy } from 'lodash';
+
+/**
+ * Internal Dependencies
+ */
+import StepWrapper from 'signup/step-wrapper';
+import wpcom from 'lib/wp';
+import Card from 'components/card';
+import Image from 'components/image';
+import Button from 'components/button';
+import { recordTracksEvent } from 'state/analytics/actions';
+import { getBusinessAddress, getBusinessName } from 'state/signup/steps/business-details/selectors';
+import { saveSignupStep, submitSignupStep } from 'state/signup/progress/actions';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface Suggestion {
+	type: string;
+	list: {
+		id: string;
+		text: string;
+		image?: string;
+	};
+}
+
+interface Props {
+	flowName: string;
+	goToNextStep: ( flowName?: string ) => void;
+	positionInFlow: number;
+	signupProgress: object[];
+	stepName: string;
+}
+
+interface SiteContact {
+	placeId: string;
+	address?: string;
+	geo?: string;
+	email?: string;
+	phone?: string;
+}
+
+interface SocialLinks {
+	[key: string]: string;
+}
+
+interface State {
+	businessContact: {
+		address?: string;
+		email?: string;
+		geo?: string;
+		phone?: string;
+	};
+	businessHours: object[];
```

Comment:
This leaves things open and I think there's benefit to getting to stricter types, but fine for now.